### PR TITLE
Drop libvirtd.sh

### PR DESCRIPTION
**Pull request by crobinso**
libvirtd.sh is the entrypoint for the container image. But AFAICT kubevirt doesn't depend on it at all anymore; this container is used as the base image for the kubevirt container that uses virt-launcher as pid 1. I put more details in the second commit. But maybe I'm missing something. @rmohr I would like your feedback in particular.

### CODE REVIEW

File: augconf
Comment (andreabolognani) [inline augconf:23]: I think it might actually makes more sense to drop the whole augeas thing entirely and rely on virt-launcher dynamically creating the libvirt configurations files.

The main reason why I think that's a good idea is that some of the settings, for example `hugetlbfs_mount`, are applied only conditionally (see [libvirt_helper.go](https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/util/libvirt_helper.go#L384-L390)), and if we're always going to need to generate *some* of the configuration through code it's IMHO more maintainable to just generate all of it in a single place.

See also `log_outputs`, which is set to `2:stderr` via augeas and to `1:stderr` by virt-launcher, for more evidence that centralizing this might be a good idea.

File: Dockerfile.in
Comment (andreabolognani) [inline Dockerfile.in:31]: If you do this, which to be clear I think is a good idea, you should also tweak `README.md` at the same time to remove all examples of the `kubevirt/libvirt` container being used in a standalone fashion, and to clarify the fact that it is intended *exclusively* as the base image for KubeVirt's virt-launcher container.

File: augconf
Comment (crobinso) [reply augconf:23]: This cgroup_controllers bit is already effectively applied in libvirt_helper.go, see my commit message. So this bit is redundant. But I wanted to cover my bases and have augconf absorb relevant bits from libvirtd.sh. I'm fine with dropping this if you want

File: Dockerfile.in
Comment (crobinso) [reply Dockerfile.in:31]: Okay, I gutted the README

File: augconf
Comment (andreabolognani) [reply augconf:23]: Yeah okay, let's leave it in for now, but I really think we should gut the whole augeas thing as a follow-up and centralize libvirtd configuration in a single place, namely `libvirt_helper.go`.

File: README.md
Comment (andreabolognani) [inline README.md]: The entire "Versions to use" section is now unnecessary as well - we literally just told the user they're **not** supposed to run the container outside of KubeVirt :)

**Comment by andreabolognani** [comment]
LGTM once you drop the last bit of README. Let's hear from @rmohr before merging though.

File: augconf
Comment (rmohr) [reply augconf:23]: +1 for removing it on the long run.

**Review by rmohr — APPROVED** [review]
Works for me. Thanks.

File: README.md
Comment (crobinso) [reply README.md]: Dropped now

**Pull request merged**
